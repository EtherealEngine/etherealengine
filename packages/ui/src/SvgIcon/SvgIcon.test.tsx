import { expect } from '@jest/globals'
import { shallow } from 'enzyme'
import React from 'react'

import SVgIcon from './index'
import { Default as story } from './index.stories'

describe('SVgIcon', () => {
  it('- should render', () => {
    const wrapper = shallow(<SVgIcon {...story?.args} />)
    expect(wrapper).toMatchSnapshot()
  })
})
