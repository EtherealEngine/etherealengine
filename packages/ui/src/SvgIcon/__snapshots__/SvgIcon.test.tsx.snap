// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SVgIcon - should render 1`] = `<ForwardRef(SvgIcon) />`;
